Dockerfile

```diff
@@ -8,37 +8,45 @@
 
 
 # Dockerfile
+#
+# -------------------------------------
+# Context: Build Context
+FROM node:12.15.0-alpine3.9 as build
```

Comment:
The only downside of this, is that we'll have to keep updating this manually as the LTS version progresses.  Maybe that's a good thing, since a change could break us, but it's something we should remember.

Reply:
We could look into a better way to manage, I'd argue being explicit is better for deployment behavior compared to using `latest` for example.

Reply:
`lts` is ["Long Term Stable"](https://nodejs.org/en/about/releases/) for node vs. "Latest."  If we always use whatever is the current LTS, we're a sweet spot for maintenance long term.  If we pin to some release, it will drift, and go out of date without intervention.

The LTS line doesn't break things very often.  If we can find a way to say on that, I'd like to.

Reply:
Good point, corrected in latest and tested.